examples/with-chatbase/README.md

```diff
@@ -0,0 +1,41 @@
+# With Chatbase
+
+## Install and Run
+
+Download this example or clone [Bottender](https://github.com/Yoctol/bottender).
+
+```
+curl https://codeload.github.com/Yoctol/bottender/tar.gz/master | tar -xz --strip=2 bottender-master/examples/with-chatbase
+cd with-chatbase
+npm install
+npm run dev
+```
+
+## Idea of this example
+
+This example shows how to combine your bot with **bot analytics tool**. In this
+case, we take [Chatbase](https://chatbase.com/welcome) which is built by Google
+as an example.
+
+The following code is the main part of this example which demonstrates the way
+to connect your bot with [Chatbase](https://chatbase.com/welcome). Furthermore,
+You have better check
+[middleware guide](https://bottender.js.org/docs/Guides-Middleware) if you don't
```

Comment:
`You have better ...` sounds not so polite.
This sentence can be rewrite as
> You can check middleware guide for more information

Reply:
We should talk about express middleware here, not our middleware guide.